docs/README.md

```diff
@@ -40,5 +40,9 @@ Search app directory entries, the web (with Google Custom Search integration), o
 ## Integration with uPortal
 + [Silent Login Configuration](silent-login.md)
 
+## Developing
+
++ [On releasing](releasing.md)
```

Comment:
Maybe combine "Integration with uPortal" and "Developing" into a "External Resource" section?